dataland-framework-toolbox/inputs/sfdr/sfdr.csv

```diff
@@ -16,97 +16,100 @@ Field Identifier,Category,Sub-Category,Field Name,Tooltip - upload page,Tooltip
-102,Social,Social and employee matters,Excessive CEO pay ratio,Average ratio of the annual total compensation for the highest compensated individual to the median annual total compensation for all employees (excluding the highest-compensated individual), ,Number,"Allowed Range: [0, INF]",Percent,Extended,,,
-103,Social,Human rights,Human Rights Policy,"Do you have a human rights policy? If yes, please share the policy with us.",Existence of a human rights policy,Yes/No, , ,Extended,,,
-104,Social,Human rights,Human Rights Due Diligence,"Do you have due diligence processes to identify, prevent, mitigate and address adverse human rights impacts?","Existence of due diligence processes to identify, prevent, mitigate and address adverse human rights impacts",Yes/No, , ,Extended,,,
-105,Social,Human rights,Trafficking In Human Beings Policy,"Do you have a policy against trafficking in human beings? If yes, please share the policy with us.",Existence of a policy against trafficking in human beings,Yes/No, , ,Extended,,,
-106,Social,Human rights,Reported Child Labour Incidents,Has their been any reported child labour incident (within own operations or supply chain)?,Reported incidents of child labor within own operations or supply chain,Yes/No, , ,Extended,,,
-107,Social,Human rights,Reported Forced Or Compulsory Labour Incidents,Has their been any reported forced or compulsory labour incident (within own operations or supply chain)?,Reported incidents of forced or compulsory labor within own operations or supply chain.,Yes/No, , ,Extended,,,
-108,Social,Human rights,Number Of Reported Incidents Of Human Rights Violations,Number of cases of severe human rights issues and incidents connected to the company, ,Integer,"Allowed Range: [0, INF]", ,Extended,,,
-109,Social,Anti-corruption and anti-bribery,Cases of Insufficient Action against Bribery and Corruption,Identified insufficiencies in actions taken to address breaches in procedures and standards of anti-corruption and anti-bribery, ,Integer,"Allowed Range: [0, INF]", ,Extended,,,
-110,Social,Anti-corruption and anti-bribery,Reported Convictions Of Bribery and Corruption,Number of reported convictions for violations of anti-corruption and anti-bribery laws, ,Integer,"Allowed Range: [0, INF]", ,Extended,,,
-111,Social,Anti-corruption and anti-bribery,Total Amount Of Reported Fines Of Bribery and Corruption,Amount of fines for violations of anti-corruption and anti-bribery laws, ,Currency,"Allowed Range: [0, INF]", ,Extended,,,
+18,Environmental,Greenhouse gas emissions ,Scope 4 GHG emissions,Scope 4 carbon emissions refer to emissions avoided when a product is used as a substitute for other goods or services that fulfill the same functions but have a lower carbon intensity., ,Number,,Tonnes,Extended,,,
```

Comment:
Anführungszeichen für den Text "Scope 4 carbon emmisions refer to ..."

Reply:
Die Anfürhungszeichen werden durch die Framework Toolbox automatisch gesezt, wenn in dem Text, wie hier zB in dem Tooltip, ein Komma enthalten ist. Das Komma trennt normalerweise die Excel Einträge in einer Zeile. Wenn wir noch ein Komma in dem Tooltip selbst haben, wird es in "" Zeichen gesetzt.